Lib/test/test_sqlite3/test_userfunctions.py

```diff
@@ -223,6 +224,14 @@ def test_func_error_on_create(self):
         with self.assertRaises(sqlite.OperationalError):
             self.con.create_function("bla", -100, lambda x: 2*x)
 
+    def test_func_too_many_args(self):
+        category = sqlite.SQLITE_LIMIT_FUNCTION_ARG
+        msg = "too many arguments on function"
+        with cx_limit(self.con, category=category, limit=1):
+            with self.assertRaisesRegex(sqlite.OperationalError, msg):
+                self.con.create_function("toomany", 2, lambda: "")
```

Comment:
Is it correct? The function has no parameters but added as a 2-parameters function.

Reply:
It does not matter, because SQLite will prevent the dummy function from running. But I agree that it looks weird like this. I'll fix it so it looks better.